adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m

```diff
@@ -77,7 +77,7 @@ GADVersionNumber GADMAdapterFyberVersionFromString(NSString *_Nonnull versionStr
     builder.spotID = spotID;
     builder.timeout = 10;
     builder.userData = extras.userData;
-    builder.muteAudio = extras.muteAudio;
+    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
```

Comment:
@2energycell @TomerFyber 
Can you confirm if this is the only place we need to update to use `IASDKCore.sharedInstance`?

Reply:
@2energycell @TomerFyber 
Gentle ping on the above comment.

Also, we can move this line before building the request (i.e. line 75).
We should probably check if the `muteAudio` is set before setting it in the `sharedInstance`:
```
  if (extras.muteAudio) {
    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
  }
 ```

Reply:
@2energycell @TomerFyber Any update on the above comments?

Reply:
@2energycell @TomerFyber Gentle bump on the above comments.